core/build.gradle

```diff
@@ -34,6 +34,10 @@ repositories {
     mavenCentral()
 }
 
+checkstyleTest.ignoreFailures = true
+checkstyleMain.ignoreFailures = true
```

Comment:
You can modify checkstyle config in [`config/checkstyle`](https://github.com/Bit-Quill/opensearch-project-sql/tree/main/config/checkstyle) to avoid conflicts and keep both active

Reply:
We decided to just ignore failures until spotless is implemented entirely for the module. Shouldn't be too long between first PR and last PR for each module.